netbox/extras/middleware.py

```diff
@@ -56,11 +60,16 @@ def __call__(self, request):
         # Process the request
         response = self.get_response(request)
 
-        # Record object changes
+        # Perform change logging and fire Webhook signals
         for obj, action in _thread_locals.changed_objects:
-            if obj.pk:
+            # Log object changes
+            if obj.pk and hasattr(obj, 'log_change'):
                 obj.log_change(request.user, request.id, action)
 
+            # Enqueue Webhooks if they are enabled
+            if settings.WEBHOOKS_ENABLED and obj.__class__._meta.verbose_name in WEBHOOK_MODELS:
```

Comment:
I think you just want to use the model name here, not the human-friendly verbose name.